.github/workflows/backup-cognito-users.yaml

```diff
@@ -0,0 +1,44 @@
+name: Create backup of cognito users
+on:
+  workflow_dispatch:
+env:
+  region: 'eu-west-1'
+  backups-bucket: 'biomage-backups-production'
+
+jobs:
+  backup-users:
+    name: create backup of cognito users
+    runs-on: ubuntu-20.04
+    steps:
+      - id: setup-node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - id: install
+        name: Install dependencies
+        run: |-
+          npm install -g cognito-backup-restore
+          sudo apt-get install jq
+
+      - id: set-up-creds
+        name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.region }}
+          
+      - id: backup-users
+        name: create backup json of cognito users
+        run: |-
+          userPoolId=$(aws cognito-idp list-user-pools --max-results 20 | jq -c '.UserPools[] | select(.Name | test("production")) | .Id')
+          cbr backup --region ${{env.region}} --key  ${{secrets.AWS_ACCESS_KEY_ID}} --secret ${{ secrets.AWS_SECRET_ACCESS_KEY }} --pool ${userPoolId} --directory .
+
+      - id: save-to-s3
+        name: save backup json to s3
+        run: |-
+          current_date=$(date '+%Y-%m-%d.%H-%M-%S')
+          userPoolName=$(aws cognito-idp list-user-pools --max-results 20 | jq -c '.UserPools[] | select(.Name | test("production")) | .Name')
+          userPoolId=$(aws cognito-idp list-user-pools --max-results 20 | jq -c '.UserPools[] | select(.Name | test("production")) | .Id')
+          aws s3api put-object --bucket ${{env.backups-bucket}} --key ${current_date}/${userPoolName}/data.json --body ./${userPoolId}.json
```

Comment:
add another step to report to slack if there was a failure like in https://github.com/hms-dbmi-cellenics/testing/blob/master/.github/workflows/run-nightly-e2e.yaml

**note** I think that the step in the nightly build tests does not work, so test that the failure notification is sent (add a wrong pool ID test name or something) before merging